src/Microsoft.Health.Fhir.Shared.Api/Features/Filters/OperationOutcomeExceptionFilterAttribute.cs

```diff
@@ -235,6 +239,22 @@ public override void OnActionExecuted(ActionExecutedContext context)
                         context.Exception = outerException;
                     }
                 }
+
+                return;
+            }
+            else
+            {
+                context.Result = CreateOperationOutcomeResult(context.Exception.Message, OperationOutcome.IssueSeverity.Error, OperationOutcome.IssueType.Unknown, HttpStatusCode.InternalServerError);
+                context.ExceptionHandled = true;
+            }
+
+            if (context.ExceptionHandled)
+            {
+                HttpStatusCode? statusCode = (context.Result as OperationOutcomeResult)?.StatusCode;
+                if (statusCode != null && statusCode >= HttpStatusCode.InternalServerError)
+                {
+                    _logger.LogError(context.Exception, "5xx error returned");
```

Comment:
>5xx [](http://example.com/codeflow?start=57&length=3)

Would it worth to make a parameter rather than just text?
I can see world where you would like to filter by error codes.